graphwalker-cli/src/main/java/org/graphwalker/cli/CLI.java

```diff
@@ -502,8 +489,8 @@ private void RunCommandOffline() throws Exception, UnsupportedFileFormat {
       contexts.get(0).setPathGenerator(GeneratorFactory.parse((String) itr.next()));
 
       if (triggerOnce &&
-          (!offline.startElement.isEmpty() ||
-           (!online.startElement.isEmpty()))) {
+        (!offline.startElement.isEmpty() ||
+          (!online.startElement.isEmpty()))) {
```

Comment:
looks like you can remove some ()

Reply:
`(!offline.startElement.isEmpty() || !online.startElement.isEmpty())` instead of `(!offline.startElement.isEmpty() || (!online.startElement.isEmpty()))`